src/store/config/types.ts

```diff
@@ -105,6 +105,7 @@ export interface GeneralConfig {
   enableDiagnostics: boolean;
   thumbnailSize: number;
   colorPickerValueRange: ColorPickerValueRange;
+  showHidden: boolean;
```

Comment:
This should not be called `showHidden` as it is just for outputs, so instead I would suggest `showHiddenOutputs`

Reply:
agree, i didnt think of that